src/main/kotlin/com/deviceinsight/helm/AbstractHelmMojo.kt

```diff
@@ -38,17 +38,20 @@ abstract class AbstractHelmMojo : AbstractMojo() {
 	@Parameter(property = "chartName", required = false)
 	private var chartName: String? = null
 
-	protected fun executeCmd(cmd: String, directory: File = target(),
-							 redirectOutput: ProcessBuilder.Redirect = ProcessBuilder.Redirect.PIPE) {
-		val proc = ProcessBuilder(cmd.split(" "))
+	protected fun executeCmd(
+		vararg cmd: String,
```

Comment:
Why not take a `List` directly? You're doing `toList` everywhere you use it anyway (except in line 59 – but that one you probably just forgot).

It would also make the spread operator in the TemplateMojo below not needed anymore and would get this function in line with its duplicate still hanging around in the LintMojo…

Reply:
I thought that the command is more ergonomic to use with varargs, but I don't have any strong feelings on this so I switched it over to using a List